autogluon/inference/docker/0.4/py3/Dockerfile.cpu

```diff
@@ -30,7 +30,7 @@ RUN pip --no-cache-dir install --upgrade --trusted-host pypi.org --trusted-host
 
 # Security patching
 RUN pip install --no-cache-dir -U 'ray>=1.10,<1.11' \
- && pip install --no-cache-dir -U 'pytorch_lightning>=1.5.10,<1.7.0' \
+ && pip install --no-cache-dir -U 'pytorch_lightning>=1.5.10,<1.6.0' \
```

Comment:
I suggest pinning setuptools version instead where setuptools is installed
```shell
pip install setuptools==59.5.0
```

Reply:
will that pin version of pytorch lightning as well?

Reply:
Please disregard the comment, I thought it was related to different issue.